cogboard-webapp/src/components/SnackbarVariantContent/styled.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "@emotion/styled/macro";
+import {SnackbarContent} from "@material-ui/core";
+
+import {mapVariantToColor} from "./helpers";
+
+export const SpanIcon = styled.span`
```

Comment:
This won't work, you have object notation for css props. With styled components use css prop notation like align-items. That's why styled is great :)